scripts/db_migrator.py

```diff
@@ -822,6 +822,21 @@ def migrate_flex_counter_delay_status_removal(self):
             self.configDB.set_entry('FLEX_COUNTER_TABLE', obj, flex_counter)
 
 
+    def migrate_frr_separate_config_mode (self):
+        """
+        Modify docker_routing_config_mode field.
+        """
+        log.log_notice('Migrate FRR Config mode configuration')
+        metadata = self.configDB.get_entry('DEVICE_METADATA', 'localhost')
+        if 'docker_routing_config_mode' not in metadata:
```

Comment:
I don't think we need to set docker_routing_config_mode explicitly if not configured since we are already assuming its unified in frr init.
We only need to add configuration when the mode is set as separate. Can you please remove the if condition?